cluster/gce/config-default.sh

```diff
@@ -467,3 +476,7 @@ ENABLE_NODE_TERMINATION_HANDLER="${ENABLE_NODE_TERMINATION_HANDLER:-false}"
 if [[ "${NODE_TERMINATION_HANDLER_IMAGE:-}" ]]; then
   PROVIDER_VARS="${PROVIDER_VARS:-} NODE_TERMINATION_HANDLER_IMAGE"
 fi
+
+# Taint Windows nodes by default to prevent Linux workloads from being
+# scheduled onto them.
+WINDOWS_NODE_TAINTS="${WINDOWS_NODE_TAINTS:-node.kubernetes.io/os=windows:NoSchedule}"
```

Comment:
We would prefer if the taint was specific to the operating system of windows since there has to be a direct correllation between host OS version and container OS version.
so the taint should be `Windows1803` in this case.
Note that 1803 is not supported by sig-windows for GA

Reply:
Yes, I'm aware of this. That's why I brought this up in the Windows GA KEP :-)
I'd like some clear recommendation of how OS version matching is going to work. For example, should it be win1809 or win2019? Is `1809` going to be compatible with `1903`? I think this may eventually lead to more confusion to the users.

BTW, this is a temporary taint we choose for GCE. We can (and will) change this later. The main purpose of this PR is to get the code in tree, so we can start testing against k8s and make incremental changes.

Reply:
I agree - for the taint name we can wait for the approach to be finalized in the KEP / elsewhere, then we can easily change this.

Reply:
Microsoft publishes a clear compatibility guide here. https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility. we document this in the KEP and don't have any ownership or influence here. it is what it is. 1809 and 1903 are incompatible for Windows server containers.

the name of the taint is not as important as long as it is consistent between containers and container host and it describes the OS version it is compatible with.